.github/workflows/kbs-e2e-azure-vtpm.yml

```diff
@@ -27,17 +27,17 @@ jobs:
       with:
         path: ./kbs.tar.gz
 
-  tdx-e2e-test:
-    needs:
-    - checkout
-    uses: ./.github/workflows/kbs-e2e.yml
-    permissions:
-      packages: write
-      contents: read
-    with:
-      runs-on-test: '["self-hosted","azure-cvm-tdx"]'
-      tee: aztdxvtpm
-      tarball: kbs.tar.gz
+  # tdx-e2e-test:
```

Comment:
@mkulke  can you add short description and probably create linked issue about commented migration to v6 ?